.github/workflows/build-documents.yml

```diff
@@ -49,3 +49,10 @@ jobs:
       - build-image
       - filters
     uses: owasp/asvs/.github/workflows/create-5.0-outputs.yml@master
+
+  publish-v5-latest:
+    if: ${{ github.event_name == 'push' && always() && needs.filters.outputs.v5 == 'true' }}
```

Comment:
I am not certain the `always()` is needed here anymore?

Reply:
Also, this will need to check that it's on the master branch, using the [`github.ref` context](https://docs.github.com/en/actions/reference/contexts-reference#github-context) -- it will probably be `github.ref == 'refs/heads/master'`